org.gnome.pushupmaster.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "org.gnome.pushupmaster",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "pushupmaster",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=/sys/bus/iio/devices/"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```

Comment:
this won't work. You need to use flatpak-builder-tools to generate a manifest for the dependencies. Or have all the crates vendored in the tarball.

https://github.com/flatpak/flatpak-builder-tools